renovate.json

```diff
@@ -1,5 +1,15 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "packageRules": [
+    {
+      "matchPackageNames": [
+        "https://github.com/redhat-appstudio/*",
+        "https://github.com/konflux-ci/*"
+      ],
+      "autoApprove": true,
+      "automerge": true
```

Comment:
the renovatebot docs recommends [using the "merge queue" together with the auto-merge functionality](https://docs.renovatebot.com/key-concepts/automerge/#if-you-use-github-actions), should we enable it too?

Reply:
I'll enable it in a different pr.